app/views/contact/foi/new.html.erb

```diff
@@ -8,7 +8,12 @@
   <%= form_for :contact, url: contact_foi_path, as: :post, authenticity_token: false, html: { class: "contact-form" } do |f| %>
     <%= hidden_field_tag 'foi[url]', Plek.new.website_root + contact_foi_path %>
 
-    <input type="text" id="val" name="foi[val]" style="display:none"/>
+    <%# Honeypot field intended to reduce spam %>
+    <div class="val" aria-hidden="true">
```

Comment:
Ah, I think I get it now. @nickcolley's comment was not just about the `aria-hidden` but there was also some CSS to go with it? (I cannot see it anymore as the commit was rebased.)
Okay, if you didn't want to use `display: none`, then doing this combination probably makes the most sense.